### DISCUSSION TITLE
How can I render just the first page of the pdf?

### DISCUSSION THREAD

**Discussion by biagiola** [diegomura/react-pdf#1151] [Q&A] [answered]
Hi. Currently I'm using the "resume" example of react-pdf. Everytime I run "yarn example resume" it throws me 6 differents types of my whole text. I just want the first page. How can I do that? o:

**Comment by Chagall** [comment, marked as answer]
The index.js file contains this part of the code which renders you 3 different styles of resumes.
```
const Output = () => (
  <Document
    author="Luke Skywalker"
    keywords="awesome, resume, start wars"
    subject="The resume of Luke Skywalker"
    title="Resume"
  >
    <Resume size="A4" />
    <Resume orientation="landscape" size="A4" />
    <Resume size={[380, 1250]} />
  </Document>
);
```

I think what you want is to render just one them, right? To do that just delete or comment 2 of the Resume objects, like this:

```
const Output = () => (
  <Document
    author="Luke Skywalker"
    keywords="awesome, resume, start wars"
    subject="The resume of Luke Skywalker"
    title="Resume"
  >
    <Resume size="A4" />
  </Document>
);
```

Then run the program again with: `yarn example -- resume `and you will get a pdf with only one format [like this](https://imgur.com/ilPjHX3)

Hope it helps, stay safe!

**Reply by biagiola** [reply]
Thanks you very much for the answer. It really helps me!!
